### PR TITLE
Fix release notes versioning and sample artifact generation

### PR DESCRIPTION
**Pull request by tyvsmith**
Description:
 - Sets task ordering for the release notes generation to enforce appropriate ordering.
 - Updated Sample apps to generate correct and buildable zip, broken last major build.gradle refactoring

Related issue(s):
#115

### CODE REVIEW

**Comment by tyvsmith** [comment]
Tested with personal repo.

File: gradle/github-release.gradle [hunk 1]
```diff
@@ -88,11 +88,31 @@ task updateNewVersionChangelog() << {
     }
 }
 
-afterReleaseBuild.dependsOn(":core-android:uploadArchives", ":rides-android:uploadArchives")
+task configureGithub() << {
```
Comment (hzsweers) [inline gradle/github-release.gradle]: nit - I think the `<<` operator is deprecated in gradle

Comment (tyvsmith) [reply gradle/github-release.gradle]: Will investigate and address at a future date.

**Comment by gbawa** [comment]
Hi Guys, 

It seems someone added me mistakenly in this repository and I can't unsubscribe. Can you guys unsubscribe me?

**Pull request merged**